src/util.h

```diff
@@ -53,6 +53,13 @@ namespace node {
 
 #define UNREACHABLE() ABORT()
 
+#define ASSIGN_OR_RETURN_UNWRAP(TYPE, PTR, OBJ, ...)                          \
+  do {                                                                        \
+    PTR = Unwrap<TYPE>(OBJ);                                                  \
```

Comment:
Another thought: consider making PTR a pointer-to-pointer, so that when you use it, it looks like this:
```c++
Foo* wrap;
ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
```
That makes it clear to humans that it's an in/out parameter.